packages/glimmer-runtime/lib/vm/frame.ts

```diff
@@ -4,19 +4,19 @@ import { InlineBlock } from '../compiled/blocks';
 import { EvaluatedArgs } from '../compiled/expressions/args';
 import { Opcode, OpSeq } from '../opcodes';
 import { LabelOpcode } from '../compiled/opcodes/vm';
-import { Component, ComponentManager, ComponentDefinition } from '../component/interfaces';
+import { Component, ComponentManager } from '../component/interfaces';
 
 class Frame {
   ops: OpSeq;
   op: Opcode;
   operand: PathReference<any> = null;
+  immediate: any = null;
```

Comment:
Its not _immediately_ obvious what `immediate` means here (pun intended), maybe an inline comment or more descriptive name?